configurations/base.py

```diff
@@ -31,7 +31,6 @@ def __new__(cls, name, bases, attrs):
         if parents:
             for base in bases[::-1]:
                 settings_vars.update(uppercase_attributes(base))
```

Comment:
I also suspect we should also bring this change, but the test suite is failing.
```suggestion
                attrs.update(uppercase_attributes(base))
```
Something we can take on later.